web/source/keyboards/kmwkeyboards.ts

```diff
@@ -485,7 +486,7 @@ namespace com.keyman {
       // Determine if the keyboard was previously loaded but is not active and use the prior load if so.
       for(Ln=0; Ln<this.keyboards.length; Ln++) { // I1511 - array prototype extended
         if(this.keyboards[Ln]['KI'] == PInternalName) {
-          keyman.textProcessor.activeKeyboard = this.keyboards[Ln];
+          keyman.textProcessor.activeKeyboard = new Keyboard(this.keyboards[Ln]);
```

Comment:
At this stage, only the currently-active keyboard will be wrapped.

In KMW, this would mean forgetting any computed local properties when swapping away from the keyboard... but at present, there aren't any.  `VKDictionary` may come soon, but it can be recomputed; stores may come after, but KMW currently persists stores to web cookies.  So, this is a reasonable compromise to take.

Reply:
What about CJK keyboards which have fairly complex code?

Reply:
CJK keyboards like to cache their lookups; they've always been doing this on the `scriptObject` directly, so this change won't break anything for them.  I've directly tested this branch with the Chinese picker keyboard.

Reply:
We don't reload the `scriptObject` - KMW still saves all previously loaded ones, just as before.  We just don't save their wrappers at this time, as that would require more extensive changes.